<launch>
    <param name="max-depth-demand"    value="3.0" />                  <!-- Maximum depth demand (m) -->
    <param name="over-depth"          value="4.0" />                  <!-- Maximum depth (m) - causes abort -->
    <param name="over-pitch"          value="45" />                   <!-- Maximum pitch (deg) - causes abort -->
    <param name="over-roll"           value="90" />                   <!-- Maximum roll (deg) - causes abort -->
    <param name="max-internal-temp"   value="50" />                   <!-- Maximum internal pressure vessel temp (deg) - causes abort -->
    <param name="min-motor-voltage"   value="19000" />                <!-- Minimum Motor voltage (mV) - causes abort -->
    <param name="mission-timeout"     value="15" />                   <!-- Mission timeout (minutes) -->
    <param name="lat_orig"            value="54.4300183333" />           <!-- Origin Latitude -->
    <param name="long_orig"           value="-7.683690" />              <!-- Origin Longitude -->
    <param name="ImportWaypoints"     value="False" />                <!-- Import Waypoints -->
    <param name="look_ahead_distance" value="10" />                   <!-- look ahead distance (in m) for track following algorithm  effectivly inverse of gain-->    
    
    
    <!--Notes-->
    <!--Testwood lat_orig=50.93642234975332 long_orig=-1.507850997377956 -->
    <!--Based on this origin X=12 and Y=15 is a good location for HOME at Testwood -->
    
    <!--Easleigh South lat_orig=50.95646666666666 long_orig=-1.366823 -->
    <!--Coordinates are for south pier, middle south of lake (X,Y) = (-32.3,22.6) -->
    
    <!--Easleigh North lat_orig=50.95702499999999 long_orig=-1.36675104 -->
    <!--Coordinates are for south pier, middle south of lake (X,Y) = (-24.6,30.0) -->
    
    <!--Manor House stone jetty.  Lat: 54.4300183333000, Long:	-7.68369000000000 -->

    
	
    <node name="Simulation_setpoints" pkg="DelphinROSv2" type="simulation_setpoints.py" />
 
    <node name="Simulator" pkg="DelphinROSv2" type="simulation.py" output="screen"  /> 
    
    <node name="heading_controller" pkg="DelphinROSv2" type="heading_controller.py"/>
    
    <node name="depthandspeed_controller_MPC_v6" pkg="DelphinROSv2" type="depthandspeed_controller_MPC_nonlinear_v10.py"/>
        
    <node name="logger" pkg="DelphinROSv2" type="logger.py" output="screen"/> 
    <!-- <node name="smach_viewer" pkg="smach_viewer" type="smach_viewer.py" /> -->

    

</launch>
